cmd/eksctl-anywhere/cmd/copypackages.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/manifests/bundles"
+	"github.com/aws/eks-anywhere/pkg/registry"
+	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+// copyPackagesCmd is the context for the copy packages command.
+var copyPackagesCmd = &cobra.Command{
+	Use:          "packages",
+	Short:        "Copy package images and charts from a source to a destination",
```

Comment:
```suggestion
	Short:        "Copy curated package images and charts from a source to a destination",
```
I think the added "curated" might be worthwhile here, since the term "package" is so generic.

Reply:
The same might go for the `Long` member. Up to you. 😄